bulk_course_settings/management/commands/process_bulk_course_settings_jobs.py

```diff
@@ -13,6 +13,8 @@
 from icommons_common.models import Term
 
 logger = logging.getLogger(__name__)
+# set to 120 seconds
+VISIBILITY_TIMEOUT = 120
```

Comment:
Might be good to have the visibility timeout be configurable, with a default value of 120.